src/dcm230/dcm230.py

```diff
@@ -119,13 +119,39 @@ class Dcm230:
     dynamically mapped to @property accessors based on `_register_specs`.
     """
 
+    BACKLIT_OPTIONS: ClassVar[list[int]] = [0, 5, 10, 20, 30, 60]
+
     SINGLE_REGISTER = 1
     MAX_REGS = 2
+
     INPUT_REGISTER = 0x03
     HOLDING_REGISTER = 0x04
 
+    DCM230_REGISTER_RESET_MAX_DMD_AND_PARTIAL_ENERGY = 0xF010
+    DCM230_REGISTER_BACKLIT_TIME = 0x003C
+    RESET_MAX_DMD = 0x0000
+    RESET_PARTIAL_ENERGY = 0x0003
+
     _register_specs: Final[dict[str, RegisterSpec]] = {
         "V": RegisterSpec(address=0x0000, count=2, decimals=1, reg_type=0x03),
+        "A": RegisterSpec(address=0x0006, count=2, decimals=1, reg_type=0x03),
+        "W": RegisterSpec(address=0x000C, count=2, decimals=1, reg_type=0x03),
+        "kwh": RegisterSpec(address=0x0048, count=2, decimals=1, reg_type=0x03),
+        "W_dmd": RegisterSpec(address=0x0054, count=2, decimals=1, reg_type=0x03),
+        "W_dmd_peak": RegisterSpec(address=0x0056, count=2, decimals=1, reg_type=0x03),
+        "kwh_tot": RegisterSpec(address=0x0156, count=2, decimals=1, reg_type=0x03),
+        "kwh_partial": RegisterSpec(address=0x0180, count=2, decimals=1, reg_type=0x03),
```

Comment:
You have constants for `INPUT_REGISTER` & `HOLDING_REGISTER`, we should use them here when setting reg_type.